tests/test_table.py

```diff
@@ -875,6 +875,18 @@ def test_insert(self, reset_db) -> None:
             t.insert(c5=np.ndarray((3, 2)))
         assert 'expected ndarray((2, 3)' in str(exc_info.value)
 
+        # test that insert with a bad column succeeds when the
```

Comment:
Wording: this isn't really about a 'bad' column, we just want to make sure that it's not trying to populate columns that aren't part of the current schema.

Reply:
done.